### ISSUE TITLE
use semantic versioning in FMF

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Risehat/freemedforms#208]
```
see http://semver.org for semantic versioning. FMF uses most of the things 
mentioned there, but the e.g. 'alpha', 'beta' etc and other special forms are 
different. 
```

Original issue reported on code.google.com by `christian.a.reiter@gmail.com` on 9 Nov 2012 at 5:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `christian.a.reiter@gmail.com` on 9 Nov 2012 at 10:28
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Pretty good definition of versioning better than mine :D
Keep it (even if we do not have a stable API right now :D)
```

Original comment by `eric.mae...@gmail.com` on 19 Nov 2012 at 10:29

**Comment by GoogleCodeExporter** [comment]
Original comment by `eric.mae...@gmail.com` on 2 Aug 2014 at 11:19
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 04a5a200bb68.
```

Original comment by `eric.mae...@gmail.com` on 3 Aug 2014 at 10:20
